python/ctsm/site_and_regional/tower_site.py

```diff
@@ -383,7 +381,7 @@ def run_case(
                 case.set_value("RUN_REFDATE", "0018-01-01")
                 case.set_value("RUN_STARTDATE", "0018-01-01")
                 case.set_value("RESUBMIT", 1)
```

Comment:
This was created originally for NEON cases, which typically start in 2018.  AD cases, would start in 0018, followed by postAD in 1018.  Maybe it's OK to keep doing this here for PLUMBER cases, which all start in different years?

Reply:
maybe we can add a note to this effect in the code?

Reply:
A comment has been added.